sound/soc/sof/intel/Kconfig

```diff
@@ -113,4 +113,15 @@ config SND_SOC_SOF_HDA
           Say Y if you want to enble HDA links with SOF.
           If unsure select "N".
 
+config SND_SOC_SOF_FORCE_LEGACY_HDA
+	bool "SOF force legacy hda and bypass DSP"
+	depends on SND_SOC_SOF_HDA
+	depends on SND_SOC_SOF_DEBUG
+	help
+          This forces the driver to use HDA legacy mode, and bypasses
+          the embedded DSP in HD-A controller on Intel SKL+ platforms.
+		  This is designed for debugging only.
+          Say Y if you do wan't to use this mode.
```

Comment:
too many double negations or typos?

Reply:
it's typo here, should be "Say Y if you do want to use this mode."